### PR TITLE
Feature/sl 1967/statuses endpoint

### PR DESCRIPTION
**Pull request by r-kruk-kainos**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/StatusConfigController.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.sandl.snlevents.model.response.StatusConfigResponse;
+import uk.gov.hmcts.reform.sandl.snlevents.repository.db.StatusConfigRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/status-config")
+public class StatusConfigController {
```
Comment (kpakur) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/StatusConfigController.java]: tests missing

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/service/StatusServiceManager.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.sandl.snlevents.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sandl.snlevents.model.db.Statusable;
+
+@Service
+public class StatusServiceManager {
```
Comment (radoslawlandowskikainos) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/service/StatusServiceManager.java]: Does this service make sense?

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/StatusConfigController.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    @Autowired
+    private StatusConfigRepository statusConfigRepository;
+
+    @GetMapping(path = "", produces = MediaType.APPLICATION_JSON_VALUE)
+    public List<StatusConfigResponse> getAllStatuses() {
+        return statusConfigRepository.findAll()
```
Comment (leszekgu) [inline src/main/java/uk/gov/hmcts/reform/sandl/snlevents/controllers/StatusConfigController.java]: are we sure it should be taken straight from the repository, not form somewhat cached service?

File: src/main/java/uk/gov/hmcts/reform/sandl/snlevents/service/StatusServiceManager.java (hunk 2 above)
Comment (leszekgu) [reply src/main/java/uk/gov/hmcts/reform/sandl/snlevents/service/StatusServiceManager.java]: If later on there will be more complicated methods inside

**Pull request merged**